src/OrchardCore.Themes/TheAdmin/wwwroot/fonts/material-design-icons/material-icons.css

```diff
@@ -0,0 +1,41 @@
+/*
+** NOTE: This file is generated by Gulp and should not be edited directly!
+** Any changes made directly to this file will be overwritten next time its asset group is processed by Gulp.
+*/
+
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(MaterialIcons-Regular.eot); /* For IE6-8 */
+  src: local('Material Icons'),
+       local('MaterialIcons-Regular'),
+       url(MaterialIcons-Regular.woff2) format('woff2'),
+       url(MaterialIcons-Regular.woff) format('woff'),
+       url(MaterialIcons-Regular.ttf) format('truetype');
```

Comment:
Looks great.

Does it include all the icons? Here we have outlined, and twotone etc
`.SetCdn("https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp")`

and this is just Regular? But I don't know what's in regular, so just checking ;)

Reply:
Ok made some changes to support all icon styles.